### DISCUSSION TITLE
2 oder mehr Nachkommastellen

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#15] [Q&A]
Hej,

weißt du wie ich die Nachkommastellen anpassen kann? Bei Temp. und Luftfeuchtigkeit werden 2 Nachkommastellen mit ausgelesen. (Siehe Bilder)

VG

![image](https://user-images.githubusercontent.com/85995176/213753876-d60f38a7-dc70-4974-b67c-951fe59e60c1.png)
![image](https://user-images.githubusercontent.com/85995176/213753724-e3d621f2-e53a-4b70-929d-5f690ae59e3f.png)

**Comment by mcuiobroker** [comment]
Steht das nicht in der Doku?

**Reply by Mephisto090** [reply]
doch bestimmt. Ich sehe da bloß leider nicht durch :(

**Comment by mcuiobroker** [comment]
![image](https://user-images.githubusercontent.com/84818934/213779628-fb95d7be-dcfe-4bc1-8d47-4467b285dc8b.png)

**Reply by Mephisto090** [reply]
![image](https://user-images.githubusercontent.com/85995176/213788743-f83f4ec0-27e7-4f21-90e1-7c0fa8476cd1.png)

irgendwie hat er keine Lust die zweite Nachkommastelle richtig zu lesen :)

**Reply by Mephisto090** [reply]
Die sind alle nicht glatt

**Reply by mcuiobroker** [reply]
Hast du ein Beispiel ioBroker-DP bei dem die Nachkommastelle anders ist?

**Reply by mcuiobroker** [reply]
Sonst evtl. ein Issue aufmachen?

**Reply by Mephisto090** [reply]
![image](https://user-images.githubusercontent.com/85995176/213793003-4d451887-c90b-4d50-8e2a-c189ef95184b.png)
![image](https://user-images.githubusercontent.com/85995176/213793144-bfd091fc-aa94-497b-8c94-e96bd58e7ad1.png)

das ist der DP zu Jarvis

**Comment by mcuiobroker** [comment]
Was hast du denn in Datenpunkt-Eigenschaften eingegeben?

![image](https://user-images.githubusercontent.com/84818934/213794144-d82240a2-a30c-424b-b6be-ea3261441667.png)
![image](https://user-images.githubusercontent.com/84818934/213794184-5346d8fc-65a3-4ad5-8b79-cce984d7347d.png)
Habe es auch mal auf role "value.temperature" angepasst und trotzdem ok
![image](https://user-images.githubusercontent.com/84818934/213795254-49ba4c70-5d28-4603-988f-5dbaa2aa669f.png)

**Reply by Mephisto090** [reply]
das aus dem Docs {"value":"val => parseFloat(val).toFixed(2)"}

**Reply by Mephisto090** [reply]
sieht nach Issue aus

**Reply by mcuiobroker** [reply]
Was ist das denn für ein Type bei dem Objekt?

![image](https://user-images.githubusercontent.com/84818934/213794470-3619aba6-88a0-4b0d-b520-e275313fd546.png)
Version `v3.1.3-beta.4`?

**Reply by Mephisto090** [reply]
"common": {
    "name": "Temperature 52 temperature",
    "type": "number",
    "role": "value.temperature",
    "read": true,
    "write": false,
    "unit": "°C",
    "def": 0,

**Reply by Mephisto090** [reply]
Ja v3.1.3-beta.4

**Reply by mcuiobroker** [reply]
Hast mal ein Upload gemacht?
`iob upload jarvis`

**Reply by Mephisto090** [reply]
> Hast mal ein Upload gemacht? `iob upload jarvis`

was meinst damit?

**Reply by mcuiobroker** [reply]
Auf der console den Befehl absetzen.

**Reply by Mephisto090** [reply]
Jetzt ja. Hat sich nichts geändert

**Reply by mcuiobroker** [reply]
Na dann, Issue hast du ja schon gemacht.

**Reply by Mephisto090** [reply]
zeigt er es bei dir richtig an, mit deinen Sensoren?

**Reply by mcuiobroker** [reply]
Ja, habe ich ja oben gezeigt.

**Reply by mcuiobroker** [reply]
Vielleicht mal WEB-Adapter neu starten?
Oder Browserdaten löschen?
